app/forms/supply_teacher_form.rb

```diff
@@ -0,0 +1,34 @@
+class SupplyTeacherForm < Form
+  attribute :employed_as_supply_teacher, :boolean
+
+  validates :employed_as_supply_teacher,
+    inclusion: {
+      in: [true, false],
+      message: ->(object, _) { object.i18n_errors_path("select_employed_as_supply_teacher") }
+    }
+
+  def initialize(claim:, journey:, params:)
+    super
+
+    self.employed_as_supply_teacher = permitted_params.fetch(
+      :employed_as_supply_teacher,
+      claim.eligibility.employed_as_supply_teacher
```

Comment:
Suggestion: given that in the next iteration we intend to avoid creating a `Claim` object during the journey I would be inclined to avoid relying on it in the form objects. To make it easier to swap out in the future we added an `update!` method in the base form class. Perhaps a convenience method in the base class to read previous answers would be helpful and allow us to swap out the underlying data source more easily.